Source/WebCore/rendering/RenderIterator.h

```diff
@@ -140,8 +181,39 @@ inline RenderObject* nextSkippingChildren(RenderObject& current, const RenderObj
     return nextAncestorSibling(current, stayWithin);
 }
 
+} // namespace WebCore::RenderObjectTraversal
+
+namespace RenderObjectPostOrderTraversal {
+
+inline RenderObject* firstLeafChild(RenderObject& object)
+{
+    return object.firstLeafChild();
+}
+
+inline RenderObject* nextAncestor(RenderObject& current, const RenderObject* stayWithin)
```

Comment:
Not sure I would have added this helper function. I would have just put this logic into the `next` function.